ros2launch/ros2launch/api/api.py

```diff
@@ -153,9 +153,12 @@ def launch_a_launch_file(
 
     # If 'launch-prefix' launch file argument is also provided in the user input,
     # the 'launch-prefix' option is applied since the last duplicate argument is used
-    if args is not None and args.launch_prefix is not None and len(args.launch_prefix) > 0:
+    if args and args.launch_prefix:
         launch_file_arguments.append(f'launch-prefix:={args.launch_prefix}')
 
+    if args.launch_prefix_filter is not None and len(args.launch_prefix_filter):
```

Comment:
@camm73 the same comment I made above applies here too:
```suggestion
    if args and args.launch_prefix_filter:
```

Reply:
Good catch—I missed that one. Fixing now